packages/qwik/src/core/index.ts

```diff
@@ -81,7 +81,7 @@ export type { QwikDOMAttributes, QwikJSX, QwikJSX as JSX } from './render/jsx/ty
 export type { QwikIntrinsicElements } from './render/jsx/types/jsx-qwik-elements';
 export type { QwikHTMLElements, QwikSVGElements } from './render/jsx/types/jsx-generated';
 export { render2 as render } from './v2/client/dom-render';
-export { getDomContainer } from './v2/client/dom-container';
+export { getDomContainer, getQContainerElement } from './v2/client/dom-container';
```

Comment:
You are making `getQContainerElement` public. I am not sure we should do that. We will need to document it and support it. I think it is better for it to stay private.

Reply:
I have changed `getQContainerElement` from public to internal. It is okay?